src/services/mongoDomainService.ts

```diff
@@ -7,16 +8,18 @@ import { DomainService } from "./domainService";
 //Remake to MONGODomainService implemented with mongo
 export class MongoDomainService extends DomainService<MongoDbService> {
   dbClient: MongoClient;
-
-  constructor(client: MongoClient, service: MongoDbService) {
+  logger: Logger;
```

Comment:
Almost, but not this is code smelling the Azure Function logger into the MongoDomainService